istioctl/pkg/writer/ztunnel/configdump/workload.go

```diff
@@ -141,18 +139,14 @@ func (c *ConfigWriter) retrieveSortedWorkloadSlice() (*ztunnelDump.ZtunnelDump,
 	return workloadDump, nil
 }
 
-func waypointName(wl *ztunnelDump.ZtunnelWorkload, prevAddresses map[string]string, workloads []*ztunnelDump.ZtunnelWorkload) string {
-	if len(wl.WaypointAddresses) == 0 {
+func waypointName(wl *ztunnelDump.ZtunnelWorkload, services map[string]*ztunnelDump.ZtunnelService) string {
+	if wl.Waypoint == nil {
```

Comment:
Can `wl` be nil here? Should we avoid panic?

Reply:
we're cycling through a list of verified workloads in `istioctl/pkg/writer/ztunnel/configdump/workload.go:PrintWorkloadSummary` - so I don't think it's a concern unless the print summary function changes drastically